sis-local/example/README.md

```diff
@@ -0,0 +1,12 @@
+# Example usage of streamlit_in_snowflake
+
+1. Create a secrets.toml file in this repo, as described in the parent directory README
+1. Create and activate a python 3.8 virtual environment
+1. Run the following
+
+```shell
+pip install -r requirements.txt
+python -m streamlit run sis_app.py
+```
```

Comment:
For a seamless local SiS experience, I think we should advocate for an Anaconda environment actually, which constraints to Snowflake Anaconda channel. That will prevent people from installing Streamlit components or other libraries which are not supported in SiS. What do you think? 

Example `environment.yml` and [instructions](https://github.com/snowflakedb/sis-datascience/tree/main#conda)
```
name: metrics
channels:
- snowflake
dependencies:
- python=3.8.12
- streamlit=1.13.0
- snowflake-snowpark-python
- plotly=5.9.0
- prophet=1.0.1
- tabulate=0.8.10
```

Reply:
+1 to this, especially since it's easier to pin yourself to python 3.8 this way. Though I would advocate for pip in general, I think it's worth using conda for a more seamless transition between local and SiS

Reply:
Ooh, except we don't want to have to push the streamlit_in_snowflake package to conda, so that probably won't work. Maybe just doing pip is better for this. We could add pip dependencies to the environment.yml, but I'm not sure what would happen if that environment.yml was deployed to SiS -- I think it would just ignore the ones it can't install, but it might error out. It's a bit tricky.

We could just have an environment.yml like above, and then recommend they pip install this package after installing that. That might be the best middle-ground.

Reply:
You can actually specify a pip entry within the conda environment.yml! Under a `pip` entry. See [docs](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually)

```
# run: conda env create --file environment.yml
name: sis
channels:
  - snowflake
dependencies:
  - python=3.8.*
  - pip
  - pip:
    - streamlit-in-snowflake
```

Reply:
Oh you mean the above environment.yml may not work in SiS. Good point. Then they should do it manually, right

Reply:
And we prioritize to add streamlit-in-snowflake in the Anaconda channel 😇